tests/regression_tests/filter_translations/test.py

```diff
@@ -0,0 +1,70 @@
+from itertools import permutations
+import numpy as np
+
+import openmc
+import pytest
+
+from tests.testing_harness import HashedPyAPITestHarness
+
+
+translations = list(permutations((10, 5, 0))) + list(permutations((-10, -5, 0)))
```

Comment:
I'm not really sure how doing a bunch of different translations is giving us better coverage. I would probably just have a single test here with the following meshes:
1. A regular mesh with no translation
2. A regular mesh defined at a different location but translated to be equivalent to 1
3. A rectilinear mesh with no translation
4. A rectilinear mesh defined at a different location but translated to be equivalent to 3